scalajvm/app/models/queries/QuoteQueries.scala

```diff
@@ -73,6 +73,13 @@ case class QuoteQueries(implicit session: DBSession) {
     }.map(rs => Quote(rs)).first().apply()
   }
 
+  def getRandomQuote: Option[Quote] = {
+    val q = Quote.syntax("q")
+    withSQL {
+      select(q.*).from(Quote as q).append(sqls"ORDER BY RANDOM()").limit(1)
```

Comment:
The most portable idea was to get the row count first and then either calculate the random value in the application code or to do something like `select * from q offset random() * @count limit 1`.